.github/workflows/bazeltest.yml

```diff
@@ -6,14 +6,7 @@ on:
     branches:
       - master
 
-  # Only run tests for PRs affecting C++ code.
   pull_request:
-    paths-ignore:
-    - '**.md'
```

Comment:
Should `*.md` files still be ignored?

Reply:
Sadly, Github expects "required checks" to run on _every_ PR, regardless of ignored paths. Ignoring `*.md` files here would prevent a docs-only change from being submitted.